services/scripts.py

```diff
@@ -1,7 +1,8 @@
 # Music recommendation application
 import pandas as pd
 from collections import defaultdict
-from cfg import TableIndexes
+from cfg import TableIndexes, MUSICALITY_ROWS_norm, MUSICALITY_ROWS
```

Comment:
```suggestion
from cfg import TableIndexes, MUSICALITY_ROWS_NORM, MUSICALITY_ROWS
```
if you changed the variable from `MUSICALITY_ROWS_norm` to `MUSICALITY_ROWS_NORM` in the config, you must change it in all files